front/src/actions/employeeActions.jsx

```diff
@@ -1,26 +1,2 @@
-import alt from '../alt'
-import EmployeeSource from '../sources/employeeSource'
 import { List } from 'immutable'
 
```

Comment:
Should the file contain actions? If so, we shouldn't need List or have the file end as .jsx? On the other side, if it is a view file we should probably rename it as actions is misleading